code/modules/mob/living/simple_animal/hostile/feral_cat.dm

```diff
@@ -36,3 +36,4 @@
 	health = 25
 	melee_damage_lower = 15
 	melee_damage_upper = 10
+	gold_core_spawnable = NO_SPAWN //Xenobio got eaten by this custom cat... (forgot to add this before, oops.)
```

Comment:
```suggestion
	gold_core_spawnable = NO_SPAWN
```
dont add code comments about what the code used to be, we have git history for that.